utils/swift_build_support/swift_build_support/products/tensorflow.py

```diff
@@ -193,6 +195,13 @@ def install(self, host_target):
                                  'usr', 'lib', 'swift',
                                  subdir, suffixed_lib_name))
 
+        # Add write permissions to `libtensorflow.{dylib,so}`.
+        # This is required for symbol stripping and codesigning.
+        subprocess.check_call(['chmod', '755',
```

Comment:
Please use `os.` operations.  This would make the script break on Windows.

Reply:
Done: `os.chmod(library, 0o755)`.